### ISSUE TITLE
The ARP help link is styled to look like plain text or a button

### ISSUE THREAD

**Issue by darbyemadewell** [department-of-veterans-affairs/va.gov-team#138635]
## Background/Context

The "Help" link is styled to look like plain text or a button, which can cause confusion about if and how it can be interacted with. "The ""Help"" link should be styled to look like a link.

Links should move the user to a new page or area of the page, while buttons should complete an action on the page. Since the ARP ""Help"" link redirects the user to a new page, it should remain a link."

## Severity

**Approximate Section 508 Severity**: N/A

**VA Severity**: a11y-defect-2

## Examples

<img width="1624" height="1000" alt="Image" src="https://github.com/user-attachments/assets/ed11de22-7582-4cb8-b22e-d737b91456ea" />
